### PR TITLE
Add README.Rmd with easy open code chunks

### PR DESCRIPTION
**Pull request by karaesmen**
The new README.Rmd:
1. suggests saving a permanent copy if user would like to save their changes.
2. provides two options to get started: open file or run dedicated code chunk 
3. Code chunk opens website in viewer pane and .Rmd file
-- Maybe ignore README.md on the cloud side?

Changes to other .Rmd files:
"click on the link" instructions changed to "visit the links provided'

### CODE REVIEW

**Review by juliasilge — CHANGES_REQUESTED** [review]
This PR has an `README.Rmd` but not an `README.md`; I believe it needs the second (rendered from the first???).

Can you render `README.md`?

**Comment by karaesmen** [comment]
Completely removed README.Rmd, simplified the README.md

File: README.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
-This repository contains R Markdown documents to accompany the [Get Started](https://www.tidymodels.org/start/) articles on https://www.tidymodels.org/:
+## R Markdown documents for RStudio Cloud ☁️ to accompany tidymodels articles
 
-1. [Build a model](https://www.tidymodels.org/start/models/)
-2. [Preprocess your data with recipes](https://www.tidymodels.org/start/recipes)  
-3. [Evaluate your model with resampling](https://www.tidymodels.org/start/resampling)  
-4. [Tune model parameters](https://www.tidymodels.org/start/tuning/)   
-5. [A predictive modeling case study](https://www.tidymodels.org/start/case-study/)
+This repository contains R Markdown documents to accompany the five [Get Started](https://www.tidymodels.org/start/) articles on https://www.tidymodels.org/.
```
Comment (apreshill) [inline README.md]: I think we only need 1 link to tidymodels.org here, I would probably just pick one?

Comment (karaesmen) [reply README.md]: ok removed!

**Review by juliasilge — APPROVED** [review]

**Pull request merged**